python/tests/test_highlevel.py

```diff
@@ -2364,7 +2364,9 @@ def modify(ts, func):
     def test_tree_node_edges(self):
         for ts in get_example_tree_sequences():
             edge_visited = np.zeros(ts.num_edges, dtype=bool)
-            for mapping, tree in zip(ts._tree_node_edges(), ts.trees()):
+            # for mapping, tree in zip(ts._tree_node_edges(), ts.trees()):
```

Comment:
I guess we can delete the ``_tree_node_edges`` thing now as well (not sure where that's come from?)

Reply:
In general we don't want to keep commented out bits lying around -that's what git is for!